packages/react-error-overlay/src/containers/StackFrameCodeBlock.js

```diff
@@ -14,7 +14,7 @@ import type { ScriptLine } from '../utils/stack-frame';
 import { primaryErrorStyle, secondaryErrorStyle } from '../styles';
 import generateAnsiHTML from '../utils/generateAnsiHTML';
 
-import codeFrame from 'babel-code-frame';
+import codeFrame from '@babel/code-frame';
```

Comment:
You might want to use `import { codeFrameColumns } from '@babel/code-frame';`, see https://www.npmjs.com/package/@babel/code-frame#upgrading-from-prior-versions